### DISCUSSION TITLE
Giving Mastodon codenames

### DISCUSSION THREAD

**Discussion by koyuawsmbrtn** [mastodon/mastodon#27906] [General]
What about giving each major Mastodon release code names? I mean even big software projects like Debian or Android do this, too.

**Comment by puskuruk** [comment]
I think it can be more clear for who looking for first time to Mastodon.

**Comment by shleeable** [comment]
`Mastodon 3.0: this time it's personal`.

**Comment by Gargron** [comment]
Mastodon 1.0 was code-named "Nipple Tooth"

https://github.com/tootsuite/mastodon/releases/v1.0

There was no such code name for 2.0

**Comment by jazmichaelking** [comment]
Looks like 3.0 should be "Chilgatherium" - https://en.wikipedia.org/wiki/Proboscidea
